### ISSUE TITLE
Cannot develop or build with v2

### ISSUE THREAD

**Issue by sudazzle** [gatsbyjs/gatsby#6669]
I ran:
```
gatsby new my-default-project https://github.com/gatsbyjs/gatsby-starter-default#v2
cd my-default-project
npm run develop
```

And It cannot develop the app. It happened both in ubuntu (bionic-beaver) and macbook (High Sierra). I am not sure what went wrong.

gatsby-starter-default@1.0.0 develop /Users/sudaman/varnish/my-default-project
> gatsby develop

success open and validate gatsby-config — 0.008 s
success load plugins — 0.176 s
success onPreInit — 0.124 s
success delete html and css files from previous builds — 0.005 s
success initialize cache — 0.009 s
success copy gatsby files — 0.029 s
success onPreBootstrap — 0.001 s
success source and transform nodes — 0.017 s
success building schema — 0.102 s
success createPages — 0.001 s
success createPagesStatefully — 0.039 s
success onPreExtractQueries — 0.000 s
success update schema — 0.072 s
success extract queries from components — 0.058 s
success run graphql queries — 0.037 s — 6/6 168.93 queries/second
success write out page data — 0.005 s
success write out redirect data — 0.001 s
success onPostBootstrap — 0.001 s

info bootstrap finished - 3.341 s

Error: ./.cache/default-html.js
  Module not found: Error: Can't resolve '@babel/runtime/helpers/inheritsLoose' in '/Users/sudaman/varnish/my-default-project/.cache'
  resolve '@babel/runtime/helpers/inheritsLoose' in '/Users/sudaman/varnish/my-default-project/.cache'
    Parsed request is a module
    using description file: /Users/sudaman/varnish/my-default-project/package.json (relative path: ./.cache)
      resolve as module
        looking for modules in /Users/sudaman/varnish/my-default-project/src/node_modules
          using description file: /Users/sudaman/varnish/my-default-project/package.json (relative path: ./src/node_modules)
            using description file: /Users/sudaman/varnish/my-default-project/package.json (relative path: ./src/node_modules/@babel/runtime/helpers/inheritsLoose)
              no extension
                /Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose doesn't exist
              .js
                /Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose.js doesn't exist
              .jsx
                /Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose.jsx doesn't exist
              as directory
                /Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose doesn't exist
        /Users/sudaman/varnish/my-default-project/.cache/node_modules doesn't exist or is not a directory
        /Users/sudaman/varnish/node_modules doesn't exist or is not a directory
        /Users/sudaman/node_modules doesn't exist or is not a directory
        /Users/node_modules doesn't exist or is not a directory
        /node_modules doesn't exist or is not a directory
        looking for modules in /Users/sudaman/varnish/my-default-project/node_modules
          using description file: /Users/sudaman/varnish/my-default-project/package.json (relative path: ./node_modules)
            using description file: /Users/sudaman/varnish/my-default-project/package.json (relative path: ./node_modules/@babel/runtime/helpers/inheritsLoose)
              no extension
                /Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose doesn't exist
              .js
                /Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose.js doesn't exist
              .jsx
                /Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose.jsx doesn't exist
              as directory
                /Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose doesn't exist
  [/Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose]
  [/Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose.js]
  [/Users/sudaman/varnish/my-default-project/src/node_modules/@babel/runtime/helpers/inheritsLoose.jsx]
  [/Users/sudaman/varnish/my-default-project/.cache/node_modules]
  [/Users/sudaman/varnish/node_modules]
  [/Users/sudaman/node_modules]
  [/Users/node_modules]
  [/node_modules]
  [/Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose]
  [/Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose.js]
  [/Users/sudaman/varnish/my-default-project/node_modules/@babel/runtime/helpers/inheritsLoose.jsx]
   @ ./.cache/default-html.js 1:0-66 8:2-16
   @ ./.cache/develop-static-entry.js

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gatsby-starter-default@1.0.0 develop: `gatsby develop`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gatsby-starter-default@1.0.0 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sudaman/.npm/_logs/2018-07-23T09_31_34_483Z-debug.log

**Comment by azdanov** [comment]
Were the project packages installed during `gatsby new` command?

**Comment by sudazzle** [comment]
Yes the project packages were installed. Deleted the node_modules and .cache and ran npm run develop and it works.

**Comment by brotzky** [comment]
@sudazzle I've tried the same on my machine and it worked.

Looks like you were able to solve the issue.

Closing as sometimes node/npm do some random stuff that leads to these errors.

**Comment by yowainwright** [comment]
There might be more to this issue. When I ran codemods for Gatsby Next, development was working. However, after trying to make my Jest Tests, work, I ran into the same error as stated above.

I added default `.babelrc` [here](https://next.gatsbyjs.org/docs/babel/). When I actually added the babel packages that are within the suggested `.babelrc`, to make Jest config work, I could not make the development build work.
